src/content/contributing/building_testing/_index.en.md

```diff
@@ -45,29 +47,61 @@ To run Go unit tests:
 make test
 ```
 
-### End-to-end tests
+### Multi-Cluster KIND Based Environment {#clusters}
+
+Shipyard provides a basic target that creates a KIND based multi-cluster
+environment, without any special deployment (apart from the default K8s):
+
+```
+make clusters
+```
+
+Find out more about [Shipyard's clusters target](../shipyard#clusters).
```

Comment:
We're repeating info here from the shipyard docs - I suggest to just link to it,

Shipyard provides functionality to [create a KIND-based cluster environment](../shipyard#clusters).

Reply:
The agreement in the end was to link from projects to here, to avoid duplication. The website will be tagged on release to allow coherence.